Change home page's section stylings

chauvuha opened the pull request:
Change landing page's stylings: created the two columns “Latest News from ASPC” and “The Student Life” and surrounded both with a box instead of leaving them as two separate sections to reduce white space and add contrast.

Before:
<img width="1268" alt="Screen Shot 2023-02-06 at 6 49 07 PM" src="https://user-images.githubusercontent.com/79251745/217135437-4f31cc2e-7614-452b-88f5-a5e273aa1b76.png">

After:
<img width="1434" alt="Screen Shot 2023-02-04 at 6 59 22 PM" src="https://user-images.githubusercontent.com/79251745/217135784-dc0d34b4-fb5e-47b4-894b-c3c0f4172089.png">

saatvikkher commented:
Remove highlights and drop shadows. Remove the TSL section. Make a two-column layout with Event carousel and Latest from ASPC. Add light box around Events.

chauvuha commented:
* Removed highlights and dropped shadows.
* Removed the TSL section. 
* Made a two-column layout with Event carousel and Latest from ASPC. 
* Added light box around Events. 
* Note: The event carousel is not showing in the local server so the demo picture of the light box around Events is generated by changing Inspect tab on Chrome. 

Before: 
<img width="648" alt="Screen Shot 2023-02-13 at 7 26 02 PM" src="https://user-images.githubusercontent.com/79251745/218631494-e3b33cf0-5e26-4d30-b9f1-8fd995ad3879.png">

After:
<img width="1020" alt="Screen Shot 2023-02-11 at 10 20 47 PM" src="https://user-images.githubusercontent.com/79251745/218630748-f8c3e01e-306c-4407-b43a-6baaa0236a0b.png">
<img width="1030" alt="Screen Shot 2023-02-11 at 10 19 31 PM" src="https://user-images.githubusercontent.com/79251745/218630762-e61f359b-a573-4608-8cf2-7500af981220.png">

Eshaan-Lumba reviewed (changes requested):
Add box around whole section like Latest from ASPC instead of around each event

chauvuha commented:
* Remove box inside slideshow items
* Added arrows in slideshow
* Moved two buttons down after slideshow

Before:
<img width="1259" alt="Screen Shot 2023-02-18 at 2 49 45 PM" src="https://user-images.githubusercontent.com/79251745/219902850-46f61184-71cf-4d7c-9386-22f71d660e26.png">

After:
<img width="1217" alt="Screen Shot 2023-02-18 at 2 50 43 PM" src="https://user-images.githubusercontent.com/79251745/219902890-9334972f-7244-4bf8-a2d9-2dd392a8cd0d.png">

saatvikkher commented:
looks great!
